SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "cube1vel[:,125,125].to(u.K).with_spectral_unit(u.km/u.s).quicklook()\n",
+    "cube2vel_reproj[:,125,125].quicklook()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Do a quick continuum subtraction"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: PossiblySlowWarning: This function (<function BaseSpectralCube._apply_everywhere at 0x2ad7b51e0488>) requires loading the entire cube into memory and may therefore be slow. [spectral_cube.utils]\n",
+      "WARNING: PossiblySlowWarning: This function (<function BaseSpectralCube._apply_everywhere at 0x2ad7b51e0488>) requires loading the entire cube into memory and may therefore be slow. [spectral_cube.utils]\n"
+     ]
+    }
+   ],
+   "source": [
+    "med1 = cube1vel.median(axis=0)  \n",
+    "cube1vel = cube1vel - med1\n",
```

Comment:
I suggest renaming the new cubes to indicate they are now background subtracted.